### PR TITLE
Onurbabat local

### PR DESCRIPTION
**Pull request by onurbabat**
What kind of food people expect when they visit Bethlehem?

### CODE REVIEW

File: onurbabat-bethlehem.md [hunk 1]
```diff
@@ -8,5 +8,8 @@
 ### Touristy things to do
 - Steel Stacks
 - Bethlehem Star
+- Bookstore
```
Comment (githubteacher) [inline onurbabat-bethlehem.md]: @onurbabat is there a particular bookstore that tourists go to?

Comment (onurbabat) [inline onurbabat-bethlehem.md]: Moravian Bookstore, which is one of the oldest bookstores in the world.

**Comment by RustyBee2016** [comment]
What are some bestsellers?

**Comment by onurbabat** [comment]
@RustyBee2016 Have not been there in a while. Check out their website http://shop.moravianbookshop.com/index.htm. They have a beautiful store. It gets very cozy before Christmas like the rest of the town.

**Pull request merged**